public/app/features/dashboard/panel_editor/HelpToggle.tsx

```diff
@@ -0,0 +1,21 @@
+import { Icon } from '@grafana/ui';
+import { css, cx } from 'emotion';
+import React, { useState } from 'react';
+
+const helpStyle = css`
+  margin-top: 4px;
+`;
+
+export const HelpToggle: React.FunctionComponent = ({ children }) => {
+  const [isHelpVisible, setIsHelpVisible] = useState(false);
+
+  return (
+    <>
+      <button className="gf-form-label query-keyword pointer" onClick={_ => setIsHelpVisible(!isHelpVisible)}>
```

Comment:
Could we rewrite the styles to emotion maybe instead of using the SASS class names?

Reply:
Or maybe even another way. Waht about using Button[variant=link] and then when the help is visible using the [InfoBox component](https://developers.grafana.com/ui/latest/index.html?path=/story/layout-infobox--basic) an utilizing its `onDismiss` property?

Reply:
![Dec-02-2020 17-34-34](https://user-images.githubusercontent.com/45561153/100909535-c64b7380-34c4-11eb-89bf-5bfff1d3bdc8.gif)

Is this what you mean?

Reply:
Personally I like it better when these kind of toggles are in the same place, so I can open close without moving a mouse or searching for the close button that much.

Reply:
I agree, just trying to get an understanding of what @dprokop had in mind

Reply:
Yups, when I see this now @kaydelaney @aocenas I agree we both of you.